src/commands/mods-install.ts

```diff
@@ -40,6 +40,8 @@ async function installMod(options: InstallModOptions): Promise<void> {
   const spec = source.spec;
   const spinner = ora.default("Installing your mod instance. This usually takes 3 to 5 minutes...");
   try {
+    logger.info(clc.bold(spec.displayName));
+    logger.info(spec.preinstallContent);
```

Comment:
This is markdown, right? Why aren't we nicely printing the rendered markdown?

Reply:
Fixed